crates/project-model/src/workspace.rs

```diff
@@ -366,11 +366,11 @@ impl ProjectWorkspace {
                 _ => None,
             })
             .collect();
-        let ref mut outputs = match WorkspaceBuildScripts::run_once(config, &cargo_ws, progress) {
+        let outputs = &mut (match WorkspaceBuildScripts::run_once(config, &cargo_ws, progress) {
```

Comment:
Are the extra parentheses even necessary here?

Reply:
negative. will update